Exdi/exdigdbsrv/GdbSrvControllerLib/GdbSrvControllerLib.cpp

```diff
@@ -2557,6 +2609,89 @@ class GdbSrvController::GdbSrvControllerImpl
         return monitorResult;
     }
 
+    SimpleCharBuffer GdbSrvControllerImpl::SetPhysicalReadMemoryMode()
+    {
+
+        SimpleCharBuffer monitorResult;
+        if (!monitorResult.TryEnsureCapacity(C_MAX_MONITOR_CMD_BUFFER))
+        {
+            throw _com_error(E_OUTOFMEMORY);
+        }
+
+        ConfigExdiGdbServerHelper& cfgData = ConfigExdiGdbServerHelper::GetInstanceCfgExdiGdbServer(nullptr);
+        LPCSTR pStrGdbSrvType;
+        size_t gdbSrvStrTypeLength;
+        wstring wGdbServerTarget;
+        cfgData.GetGdbServerTargetName(wGdbServerTarget);
+        using convert_type = std::codecvt_utf8<wchar_t>;
+        std::wstring_convert<convert_type, wchar_t> converter;
+        const std::string sGdbServerTarget = converter.to_bytes(wGdbServerTarget);
+        if (sGdbServerTarget.empty())
+        {
+            throw _com_error(E_FAIL);
+        }
+
+        pStrGdbSrvType = sGdbServerTarget.c_str();
+        gdbSrvStrTypeLength = sGdbServerTarget.length();
+        if (!cfgData.GetServerRequirePAMemoryAccess())

```

Comment:
Unused locals?

Reply:
Yes, I don't see any VS build spew about the unused variables.